includes/class-ckwc-integration.php

```diff
@@ -798,9 +817,6 @@ public function enqueue_scripts() {
 						'sync_past_orders_confirmation_message' => __( 'Do you want to send past WooCommerce Orders to Kit?', 'woocommerce-convertkit' ),
 					)
 				);
-
-				// Enqueue Select2 JS.
-				ckwc_select2_enqueue_scripts();
```

Comment:
This was never needed, as we didn't use select2 on the settings screen prior to this PR. We now use it, but WooCommerce loads its own version of the select2 library for us.